js/src/components/google-combo-account-card/account-creation-description/index.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import useGoogleAccount from '.~/hooks/useGoogleAccount';
+import useGoogleMCAccount from '.~/hooks/useGoogleMCAccount';
+import useGoogleAdsAccount from '.~/hooks/useGoogleAdsAccount';
+import AccountInfo from './account-info';
+
+/**
+ * Renders the description for the account creation card.
+ *
+ * @param {Object} props Props.
+ * @param {Object} props.accountsCreated Whether accounts have been created.
+ * @param {boolean} props.isCreatingBothAccounts Whether both, MC and Ads accounts are being created.
+ * @param {boolean} props.isCreatingMCAccount Whether Merchant Center account is being created.
+ * @param {boolean} props.isCreatingAdsAccount Whether Google Ads account is being created.
+ */
+const AccountCreationDescription = ( {
+	accountsCreated,
+	isCreatingBothAccounts,
+	isCreatingMCAccount,
+	isCreatingAdsAccount,
+} ) => {
+	const { google } = useGoogleAccount();
+	const {
+		googleMCAccount,
+		hasFinishedResolution: hasFinishedResolutionForCurrentMCAccount,
+	} = useGoogleMCAccount();
+
+	const {
+		googleAdsAccount,
+		hasFinishedResolution: hasFinishedResolutionForCurrentAdsAccount,
+	} = useGoogleAdsAccount();
+
+	const isLoadingData =
+		accountsCreated &&
+		( ! hasFinishedResolutionForCurrentMCAccount ||
+			! hasFinishedResolutionForCurrentAdsAccount );
+
+	const getDescription = () => {
+		if (
+			isLoadingData ||
+			isCreatingBothAccounts ||
+			isCreatingMCAccount ||
+			isCreatingAdsAccount
```

Comment:
I've removed `isLoadingAccountsData` from the `if` block, but we will still need to check 

```js
isCreatingBothAccounts || isCreatingMCAccount || isCreatingAdsAccount
```

because we want to display different processing messages based on which account is being created.

`isLoadingAccountsData` will be used to display the Merchant account and Ads account information and will be displayed once the data is loaded.

Reply:
@eason9487 This has been changed and now component accepts `isCreatingWhichAccount` prop which is one of the following.

- `null` - When no accounts are being created.
- `mc` - If MC account is being created.
- `ads` - If Ads account is being created. 
- `both` - If both accounts are being created.